scripts/longtermeffects.lua

```diff
@@ -252,35 +228,20 @@ local function nextRound_new(nRounds, bTimeChanged)
 	end
 end
 
-local function clearExpiringEffects_new(bShort)
-end
-
 -- Function Overrides
 function onInit()
 	local sRuleset = User.getRulesetName()
-	if sRuleset == '3.5E' or sRuleset == 'PFRPG' or sRuleset == 'PFRPG2' or sRuleset == '5E' then
-		nextRound_old = CombatManager.nextRound;
+	if sRuleset == "3.5E" or sRuleset == "PFRPG" or sRuleset == "PFRPG2" or sRuleset == "5E" then
 		CombatManager.nextRound = nextRound_new;
-		
-		resetInit_old = CombatManager.resetInit;
-		CombatManager.resetInit = resetInit_new;
-		
-		clearExpiringEffects_old = CombatManager2.clearExpiringEffects;
-		CombatManager2.clearExpiringEffects = clearExpiringEffects_new;
-
+		CombatManager.setCustomCombatReset(onCustomCombatReset);
 		EffectManager.setCustomOnEffectAddStart(onEffectAddStart_new);
-		if sRuleset == '3.5E' or sRuleset == 'PFRPG' then
+		if sRuleset == "3.5E" or sRuleset == "PFRPG" then
 			EffectManager35E.onEffectAddStart = onEffectAddStart_new;
-		elseif sRuleset == '5E' then
+		elseif sRuleset == "5E" then
 			EffectManager5E.onEffectAddStart = onEffectAddStart_new;
 		end
 	end
 
-	OptionsManager.registerOption2('TIMEROUNDS', false, 'option_header_game', 'opt_lab_time_rounds', 'option_entry_cycler', { labels = 'enc_opt_time_rounds_slow', values = 'slow', baselabel = 'enc_opt_time_rounds_fast', baseval = 'fast', default = 'fast' });
-end
-
-function onClose()
-	CombatManager.nextRound = nextRound_old;
-	CombatManager.resetInit = resetInit_old;
-	CombatManager2.clearExpiringEffects = clearExpiringEffects_old;
+	OptionsManager.registerOption2("TIMEROUNDS", false, "option_header_CLOCKADJUSTER", "opt_lab_time_rounds", "option_entry_cycler",
+		{ labels = "enc_opt_time_rounds_slow", values = "slow", baselabel = "enc_opt_time_rounds_fast", baseval = "fast", default = "fast" });
```

Comment:
Move this option to the new Clock Adjuster section.  I don't think the Full Process feature works though... consider removing it, it wasn't advancing the number of rounds for whatever I added to the clock.